CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ $ cd gvm-tools && git log
 * Added `pretty_print` to `pyshell` by default, so it does not need to be manually imported [#305](https://github.com/greenbone/gvm-tools/pull/305)
 * Added tests for `helper` module [#310](https://github.com/greenbone/gvm-tools/pull/310)
 * Added tests for `parser` module [#311](https://github.com/greenbone/gvm-tools/pull/311)
+* Added test for `script/send-targe.gmp.py` [#314](https://github.com/greenbone/gvm-tools/pull/314/)
```

Comment:
```suggestion
* Added test for `script/send-target.gmp.py` [#314](https://github.com/greenbone/gvm-tools/pull/314/)
```

Reply:
Sorry I guess i've introduced this typo

Reply:
yeah. but I have just commited your suggestion ...